packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -275,6 +276,18 @@ class MenuAnchor extends StatefulWidget {
   /// {@macro flutter.material.MenuBar.shortcuts_note}
   final List<Widget> menuChildren;
 
+  /// The optional alignment for the widget that this [MenuAnchor] surrounds.
+  ///
+  /// If provided, [builder] is placed inside an [Align] using this alignment.
+  /// This can be used when the [MenuAnchor] parent sets tight constraints
+  /// which will result in the menu appearing below these tight constraints
+  /// instead of below the widget returned by the [builder].
+  ///
+  /// The recommanded value is [AlignmentDirectional.topStart].
+  ///
+  /// Defaults to null which means no [Align] is added by default.
```

Comment:
I _think_ (could be wrong) there needs to be a comma after null because "which means no [Align] is added by default" is a nonrestrictive clause.